src/main/java/com/minecolonies/coremod/achievements/ModAchievements.java

```diff
@@ -39,279 +38,267 @@
+    public static final MineColoniesAchievement achievementHarvest1000Potatoes = new MineColoniesAchievement("farmer.potatoesHarvested.1000", 12, 16, Items.POTATO,
+            achievementHarvest500Potatoes);
+    public static final MineColoniesAchievement achievementHarvest1000Wheat    = new MineColoniesAchievement("farmer.wheatHarvested.1000", 12, 17, Blocks.HAY_BLOCK,
+            achievementHarvest500Wheat);
 
     // Sizes
     /**
      * Reach {@link ModAchievements#ACHIEVEMENT_SIZE_SETTLEMENT} citizens.
      */
-    public static final Achievement achievementSizeSettlement = new MineColoniesAchievement("size.settlement", 4, 1, ModItems.itemAchievementProxySettlement,
-                                                                                             achievementTownhall).registerStat();
+    public static final MineColoniesAchievement achievementSizeSettlement = new MineColoniesAchievement("size.settlement", 4, 1, ModItems.itemAchievementProxySettlement, achievementTownhall);
     /**
      * Reach {@link ModAchievements#ACHIEVEMENT_SIZE_TOWN} citizens.
      */
-    public static final Achievement achievementSizeTown       = new MineColoniesAchievement("size.town", 6, 1, ModItems.itemAchievementProxyTown, achievementSizeSettlement)
-                                                                  .registerStat();
+    public static final MineColoniesAchievement achievementSizeTown       =
+            new MineColoniesAchievement("size.town", 6, 1, ModItems.itemAchievementProxyTown, achievementSizeSettlement);
     /**
      * Reach {@link ModAchievements#ACHIEVEMENT_SIZE_CITY} citizens.
      */
-    public static final Achievement achievementSizeCity       = new MineColoniesAchievement("size.city", 8, 1, ModItems.itemAchievementProxyCity, achievementSizeTown)
-                                                                  .registerStat();
+    public static final MineColoniesAchievement achievementSizeCity       = new MineColoniesAchievement("size.city", 8, 1, ModItems.itemAchievementProxyCity, achievementSizeTown);
 
     /**
      * Reach {@link ModAchievements#ACHIEVEMENT_SIZE_METROPOLIS} citizens.
      */
-    public static final Achievement achievementSizeMetropolis = new MineColoniesAchievement("size.metropolis", 10, 1, ModItems.itemAchievementProxyMetropolis, achievementSizeCity)
-                                                                  .registerStat();
+    public static final MineColoniesAchievement achievementSizeMetropolis = new MineColoniesAchievement("size.metropolis", 10, 1, ModItems.itemAchievementProxyMetropolis, achievementSizeCity);
 
-    // Achievement pages
+    // MineColoniesAchievement pages
     /**
      * The MineColonies achievement page.
      */
-    private static final AchievementPage achievementPageMineColonies = new MineColoniesAchievementPage(
+    private static final MineColoniesAchievementPage achievementPageMineColonies = new MineColoniesAchievementPage(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "achievementPageMineColonies" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)